features/admin.connections.v1/utils/connection-utils.ts

```diff
@@ -428,6 +428,33 @@ export const handleGetConnectionsError = (error: AxiosError): void => {
     return;
 };
 
+/**
+ * Utility function to handle the error alert of the get custom authenticator request.
+ */
+export const handleGetCustomAuthenticatorError = (error: AxiosError): void => {
+    if (error?.response?.data?.description) {
+        store.dispatch(
+            addAlert({
+                description: I18n.instance.t("authenticationProvider:notifications.getIDP.error.description", {
```

Comment:
Don't we have a custom error message for custom authenticators.
If not can't we directly use the idp error functions ?